daemon/daemon_main.go

```diff
@@ -829,6 +829,10 @@ func init() {
 	flags.MarkHidden(option.PolicyTriggerInterval)
 	option.BindEnv(option.PolicyTriggerInterval)
 
+	flags.Bool(option.DisableCNPStatusUpdates, false, "do not send CNP NodeStatus updates")
+	flags.MarkHidden(option.DisableCNPStatusUpdates)
```

Comment:
What is the reason to mark this hidden?

Reply:
I usually hide the more under-the-hood configuration parameters by default, but am happy to change this to not be hidden. I'll re-upload.

Reply:
Removed the `MarkHidden` call for this option.